tests/test_micropipenv.py

```diff
@@ -50,6 +51,18 @@ def get_pip_path(venv):
     return os.path.join(venv.path, "bin", "pip3")
 
 
+def setup_module():
+    """A dirty hack for mypy that does not accept collisions in file names."""
+    for item in glob.glob(os.path.join(_DATA_DIR, "**", "setup"), recursive=True):
```

Comment:
```suggestion
    for item in glob.glob(os.path.join(_DATA_DIR, "**", "setup.py"), recursive=True):
```

Reply:
Yes - the pytest-mypy plugin we use otherwise reports errors on that file as it finds multiple `setup.py` in the directory structure. Sadly, I couldn't find any other way how to bypass it.

Reply:
oh, good to know.